scripts/run_docker.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash -e
+
+# The script must be called from the root of the github tree, else it returns an error.
+
+if [[ $(git rev-parse --show-toplevel) != $(pwd) ]]; then
+  echo "ERROR: This script must be started from the root of the git repo";
+  exit 1;
+fi
+
+
+source ./.env;
+export COMPOSE_FILE=${COMPOSE_FILE:-./docker-compose/docker-compose.yml};
+
+
+function setup() {
+  docker compose build;
+  docker compose up -d;
+}
+
+function teardown() {
+  rm -f cookies.txt;
+  docker compose down -v;
+  docker image rm ghcr.io/c4dt/d-voting-frontend:latest ghcr.io/c4dt/d-voting-backend:latest ghcr.io/c4dt/d-voting-dela:latest;
+  docker builder prune -f;
```

Comment:
Might be a bit aggressive here, no? Doesn't it remove all builder caches on the system?

Reply:
yes, that I should change